hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/contract/precompile/TokenInfoHTSSuite.java

```diff
@@ -1370,7 +1370,7 @@ final Stream<DynamicTest> happyPathUpdateTokenKeysAndReadLatestInformation() {
                                         HapiParserUtil.asHeadlongAddress(
                                                 asAddress(spec.registry().getContractId(TOKEN_INFO_CONTRACT))))
                                 .via(UPDATE_AND_GET_TOKEN_KEYS_INFO_TXN)
-                                .gas(117000L)
+                                .gas(118000L)
```

Comment:
Why does this change as a result of the shard and realm differences?

Reply:
zero values and non zero values can incur a different gas cost in some operations.